### PR TITLE
WT-2624 Fix example program build with MSVC 2013

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

**Comment by daveh86** [comment]
@agorrod, mind having a look at this?

File: examples/c/ex_file_system.c [hunk 1]
```diff
@@ -31,6 +31,12 @@
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
+#ifndef _WIN32
+#include <unistd.h>
```
Comment (agorrod) [inline examples/c/ex_file_system.c]: Why did you add ``unistd.h`` here? Is there a missing symbol without it?

Comment (daveh86) [inline examples/c/ex_file_system.c]: Added so that it uses the same logic as the other example files.

I don't think its needed in this case but wanted to keep the examples looking similar.

Can remove if need be.

Comment (agorrod) [inline examples/c/ex_file_system.c]: @daveh86 Yes - please make that change. Maybe copy the clause from ``ex_scope.c``

**Comment by agorrod** [comment]
Thanks @daveh86 lgtm - I'll merge.
